stellar-contract-env-common/src/env.rs

```diff
@@ -103,6 +105,7 @@ macro_rules! call_macro_with_all_host_functions {
                 {"$9", fn vec_back(v:Object) -> RawVal}
                 {"$A", fn vec_insert(v:Object, i:RawVal, x:RawVal) -> Object}
                 {"$B", fn vec_append(v1:Object, v2:Object) -> Object}
+                {"$C", fn vec_slice(v:Object, i:RawVal) -> Object}
```

Comment:
❗ Slice should have three parameters, it should have an index and a length, but it looks like it only has an index here.

Reply:
Ah, good catch, fixed! (I somehow had "vec_split" in mind when I wrote this)